mxcube3/ui/components/SampleView/SampleControls.js

```diff
@@ -1,3 +1,5 @@
+/* eslint no-unused-expressions: ["error", { "allowTernary": true }] */
```

Comment:
It is complaining about this:

"    clickCentring ? sendAbortCentring() : sendStartClickCentring() " 

I can add it globally but I think we should be allowed to use "?" operator, do you agree?